daemon/src/main/scala/io/gearpump/cluster/worker/Worker.scala

```diff
@@ -441,11 +447,14 @@ private[cluster] object Worker {
 
   private def filterDaemonLib(classPath: Array[String]): Array[String] = {
     val jarFile = new File(getClass.getProtectionDomain.getCodeSource.getLocation.toURI.getPath)
-    val daemonDir = jarFile.getParent
-    classPath.filterNot{path =>
+    val gearHome = jarFile.getParentFile.getParentFile.getParent
```

Comment:
better to move this gearHome path finding code into a standalone function and detail document the rationale and assumed dir structure.